modules/ROOT/pages/authentication-authorization/password-and-user-recovery.adoc

```diff
@@ -1,4 +1,5 @@
 :description: This page describes how to reset a password to recover a user's access when their password is lost. It specifically focuses on how to recover an admin user if all the admin users have been unassigned the admin role, and how to recreate the built-in admin role if it has been dropped.
+:page-aliases: configuration/password-and-user-recovery.adoc
```

Comment:
I prefer to move content instead of using aliases because, after some time, we might end up with many aliases and find it challenging to find the actual content.

Reply:
No, but it was moved. The alias is just in case there is another page where the old url is used (which there is), so it's redirected to the new path, so avoiding an error 404.

Reply:
Ah, ok. I didn't know that page aliases can be used instead of redirects.